src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const mainImg = document.querySelector('#largeImg');
+const thumbs = document.querySelector('#thumbs');
+
+// eslint-disable-next-line no-shadow
+thumbs.addEventListener('click', (event) => {
+  event.preventDefault();
```

Comment:
The `event.preventDefault()` call is used to prevent the default action of the event. Ensure that this is necessary for your use case. If the default action of the click event on the anchor tags is not causing any issues, you might not need this line.